lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -824,6 +839,63 @@ getConfigParam = label "getConfigParam" $ do
     secondsToNominalDiffTime =
         toEnum . fromEnum . secondsToDiffTime .  fromIntegral
 
+    getRewardParams :: Get RewardParams
+    getRewardParams = do
+        tag <- getWord8
+        case tag of
+            1 -> LinearRewards <$> getLinearParams
+            2 -> HalvingRewards <$> getHalvingParams
+            _ -> fail $ "Unknown reward parameters with tag " ++ show tag
+
+    -- NOTE: getHalvingParams and getLinearParams are currently identical.
```

Comment:
Actually, maybe we should leverage that they are identical. Jormungandr does not, but if jörmungandr differentiates them, that would be a problem for another day.

Reply:
Yes.